examples/sdk/react/src/App.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import './App.css';
+import { BacktraceClient } from '@backtrace/react';
+
+function App() {
+    const [clicked, setClicked] = useState(false);
+    function sendError() {
+        const client = BacktraceClient.instance;
+        if (client) {
+            client.send(new Error('Manual Test Error'));
+            alert('Error sent! Check your Backtrace console to see the error.');
```

Comment:
the method is asynchronous. Can we make it async? I think it would be great if we can make it sync later but you can't expect to synchronously send it to the server.

Alert is also ... easy way to demonstrate something, but not a really cool way to do that. Wouldn't it be better to use toast/div error message instead?

Reply:
async -> fixed

Also I added a toast when the error gets send and also if the submission url hasn't been updated.